plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -696,6 +698,116 @@ void GSRendererDX11::ResetStates()
 	m_om_dssel.key = 0;
 }
 
+// Partial port of OGL SW blending. Currently only works for accumulation blend or when m_sw_blending
+// m_sw_blending >= ACC_BLEND_HIGH_D3D11 since it has not been fully tested. There is no check
+// currently for primitive overlap like in OGL but it should give better results for post-processing effects
+// that are done by non-overlapping sprites.
+void GSRendererDX11::EmulateBlending()
+{
+	const GIFRegALPHA& ALPHA = m_context->ALPHA;
+	bool sw_blending         = false;
+
+	m_om_bsel.abe = PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS);
+	m_om_bsel.a = ALPHA.A;
+	m_om_bsel.b = ALPHA.B;
+	m_om_bsel.c = ALPHA.C;
+	m_om_bsel.d = ALPHA.D;
+
+	if (m_env.PABE.PABE)
+	{
+		if (m_om_bsel.a == 0 && m_om_bsel.b == 1 && m_om_bsel.c == 0 && m_om_bsel.d == 1)
+		{
+			// this works because with PABE alpha blending is on when alpha >= 0x80, but since the pixel shader
+			// cannot output anything over 0x80 (== 1.0) blending with 0x80 or turning it off gives the same result
+
+			m_om_bsel.abe = 0;
+		}
+		else
+		{
+			//Breath of Fire Dragon Quarter triggers this in battles. Graphics are fine though.
+			//ASSERT(0);
+		}
+	}
+
+
+	// Accumulation blend
+	uint8 blend_index  = uint8(((ALPHA.A * 3 + ALPHA.B) * 3 + ALPHA.C) * 3 + ALPHA.D);
+	int blend_flag = GSDevice11::m_blendMapD3D11[blend_index].flags;
+
+	bool impossible_or_free_blend =
+		(blend_flag & (BLEND_A_MAX|BLEND_NO_BAR|BLEND_ACCU))
+		|| m_ps_sel.fbmask;
```

Comment:
I think adding m_bind_rtsample instead of m_ps_sel.fbmask would work too since fbmask calls the bool anyway.

Reply:
Maybe we don't need to include Unsafe FB mask in impossible/free blend like on gl.